src/main/java/com/zotov/edu/passportofficerestservice/service/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.zotov.edu.passportofficerestservice.service.exception;
+
+public class EntityNotFoundException extends RuntimeException {
```

Comment:
`serialVersionUID` is missed.
Better to read about `Serializable` interface to get some knowledge about it.

Reply:
Added the serialVersionUID to all classes implementing serializable.